src/Lumi/Components2/Button.purs

```diff
@@ -68,7 +69,7 @@ button =
           , className: props.className
           , css:
             toCSS theme props (Styles.Button.button props.color props.kind props.state props.size)
-          , onClick: capture_ props.onPress
+          , onClick: handler_ props.onPress
```

Comment:
I wonder if there are any implications from changing this…

Reply:
Strange. Probably not because we don't often use event bubbling. Only place I can think of is table row links.